### PR TITLE
WD-12853 - rebrand /gcp/fips

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Rebrand  /gcp/fips. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14249.demos.haus/gcp/fips)
- [figma](https://www.figma.com/design/vz3yAG1nYI7r5gc3SpTUOw/ubuntu.com%2Fgcp%2Ffips?node-id=1-3103&node-type=FRAME&t=p9GnMvla4VmAGUIl-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12853](https://warthogs.atlassian.net/browse/WD-12853)

[WD-12853]: https://warthogs.atlassian.net/browse/WD-12853?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/gcp/fips.html [hunk 1]
```diff
@@ -4,354 +4,446 @@
...
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
-<section class="p-strip u-no-padding--top u-no-padding--bottom">
-  <div class="p-strip--suru-topped">
-    <div class="row u-equal-height">
-      <div class="col-7 col-medium-4">
+<section class="p-section--hero">
```
Comment (britneywwc) [inline templates/gcp/fips.html:12]: Screenshot 2024-09-03 at 12 10 48 PM

The CTA buttons are missing on hero section

File: templates/gcp/fips.html [hunk 2]
```diff
@@ -4,354 +4,446 @@
...
-          <div class="p-matrix__content">
-            <h3 class="p-matrix__title">FIPS compliant by default</h3>
-            <div class="p-matrix__desc">
+    <div class="col-start-large-4 col-9 col-medium-6">
+      <hr class="p-rule--muted" />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <div class="list-matrix">
+            <div class="list-matrix-container">
+              <p class="p-heading--5">FIPS compliant by default</p>
```
Comment (britneywwc) [inline templates/gcp/fips.html:39]: There is too much space between the list title and paragraph items
Screenshot 2024-09-03 at 12 12 51 PM

File: templates/gcp/fips.html [hunk 3]
```diff
@@ -4,354 +4,446 @@
...
+    <hr />
+    <div class="col">
       <h2>Google Cloud compliance made easy</h2>
-      <p><strong>Ubuntu Pro FIPS Google Cloud compliance made easy</strong></p>
-      <p>With Google Cloud, US government agencies and their partners have been able to combine unparalleled flexibility, breakthrough innovation, world class security and Google Cloud compliance. However when it comes to running a secure and compliant enterprise Linux on Google Cloud, IT departments are often left with expensive options that sacrifice innovation for compliance. Watch this video on demand to see how easy for government agencies to consume Linux on Google Cloud.</p>
-     <p><a href="https://ubuntu.com/blog/enable-fips-on-google-cloud" class="p-button--positive">Read the blog</a></p>
+    </div>
+    <div class="col"> 
+      <div class="p-image-container--16-9 u-hide--small is-highlighted">
+      <img class="p-image-container__image u-no-padding--top" style="width: 70%;" src="https://assets.ubuntu.com/v1/719e550b-Things%20to%20do.svg"  alt="" />
```
Comment (britneywwc) [inline templates/gcp/fips.html:191]: There should be some spacing below the image as per design
Screenshot 2024-09-03 at 12 14 29 PM

File: templates/gcp/fips.html [hunk 4]
```diff
@@ -4,354 +4,446 @@
...
-      <p><a href="https://assets.ubuntu.com/v1/8ad870d4-Ubuntu-Pro-FIPS-for-Google-Cloud-compliance-DS-14-12.22.pdf">Read the Ubuntu Pro FIPS datasheet&nbsp;&rsaquo;</a></p>
     </div>
-  </div>
 
-  <div class="u-fixed-width">
-    <hr class="p-separator">
+    <table class="p-table--mobile-card">
+      <thead>
+        <tr>
+          <th class="u-align--right" style="width: 9%;">Ubuntu LTS</th>
```
Comment (britneywwc) [inline templates/gcp/fips.html:211]: The items under "Ubuntu LTS" should be aligned to the left, similar for the table below
Screenshot 2024-09-03 at 12 15 53 PM

File: templates/gcp/fips.html [hunk 5]
```diff
@@ -4,354 +4,446 @@
...
+    <hr />
+    <div class="col">
+      <h2>Get Ubuntu Pro FIPS on Google Cloud now:</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-3">
+          <ul class="p-list--divided">
+            <li class="p-list__item" style="box-shadow: unset;">
+              <a href="https://console.cloud.google.com/compute">18.04 LTS&nbsp;&rsaquo;</a>
```
Comment (britneywwc) [inline templates/gcp/fips.html:407]: The order of LTS versions should be arranged from latest to oldest

**Comment by britneywwc** [comment]
Left some comments, could you also fix the failing CI actions?

File: templates/gcp/fips.html (hunk 1 above)
Comment (lizzochek) [reply templates/gcp/fips.html:12]: They are not on the copy doc, so shouldn't be there

File: templates/gcp/fips.html (hunk 2 above)
Comment (lizzochek) [reply templates/gcp/fips.html:39]: For now we agreed with design to use this pattern until it is upstreamed in Vanilla

File: templates/gcp/fips.html (hunk 1 above)
Comment (lizzochek) [reply templates/gcp/fips.html:12]: oh, I may have been on the wrong doc, thanks

Comment (lizzochek) [reply templates/gcp/fips.html:12]: https://docs.google.com/document/d/17Rr6Fdrr7xSPi1g2h7GMTztQUGYqz2hdbNUqK0uQihw/edit

Comment (britneywwc) [reply templates/gcp/fips.html:12]: Can you update the meta copy docs to the correct one?

Comment (lizzochek) [reply templates/gcp/fips.html:12]: done

**Pull request merged**